docs/pages/guides/dom-components.mdx

```diff
@@ -158,6 +158,59 @@ getDeviceName(): Promise<string> {
 
 Think of these functions like React Server Actions, but instead of residing on the server, they live locally in the native app and communicate with the DOM component. This approach provides a powerful way to add truly native functionality to your DOM components.
 
+## Passing refs
+
+You can use the `useDomImperativeHandle` hook inside a DOM component that to accept ref calls from the native side. The `useDomImperativeHandle` is similar to React's [`useImperativeHandle`](https://react.dev/reference/react/useImperativeHandle) hook, but it does not need a ref object to be passed to it.
```

Comment:
```suggestion
You can use the `useDomImperativeHandle` hook inside a DOM component to accept ref calls from the native side. This hook is similar to React's [`useImperativeHandle`](https://react.dev/reference/react/useImperativeHandle) hook, but it does not need a ref object to be passed to it.
```

Readability improvements: Let's remove the word "that" as it breaks the sentence while reading. Also, we don't have to repeat the hook name in the same sentence twice since we started the first sentence with the hook itself.